### DISCUSSION TITLE
Display actual nested trees in the Admin UI

### DISCUSSION THREAD

**Discussion by janus-reith** [payloadcms/payload#5663] [Feature Requests & Ideas]
For Pages it would be great to have a collapsible page tree.
The nestedDocs plugin probably already covers the database facing parts for three structures, so probably it's just a matter of visual representaton.
I think this could be a reasonable core addition since its a typical feature of many CMS Admin panels.

**Comment by sannajammeh** [comment]
I second this!

One of the reasons we can't yet more many projects over to Payload is the lack of nested folders, especially for Media objects.

**Comment by GermanJablo** [comment]
Closing because it is included in https://github.com/payloadcms/payload/discussions/4190
This feature is a WIP, so stay tuned!
